packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -201,6 +209,19 @@ export const Subscriptions = ({
     return <LoadingOverlay isLoading={true} />;
   }
 
+  if (subsequentInvoices.error !== null) {
```

Comment:
nit, could you move this section of code above the previous if statement? This way all `FetchErrorDialogMessage` follow each other.

Reply:
done